frontend/app/src/components/settings/api-keys/ExchangeKeysForm.vue

```diff
@@ -281,6 +267,59 @@ onMounted(() => {
 
 const krakenAccountTypes = KrakenAccountType.options;
 const exchanges = SUPPORTED_EXCHANGES;
+
+const sensitiveFieldEditable = computed(() => !get(edit) || get(editKeys));
+
+const rules = {
+  name: {
+    required: helpers.withMessage(
+      tc('exchange_keys_form.name.non_empty'),
+      requiredIf(computed(() => !get(edit)))
+    )
+  },
+  newName: {
+    required: helpers.withMessage(
+      tc('exchange_keys_form.name.non_empty'),
+      requiredIf(edit)
+    )
+  },
+  apiKey: {
+    required: helpers.withMessage(
+      tc('exchange_keys_form.api_key.non_empty'),
+      requiredIf(sensitiveFieldEditable)
+    )
+  },
+  apiSecret: {
+    required: helpers.withMessage(
+      tc('exchange_keys_form.api_secret.non_empty'),
+      requiredIf(sensitiveFieldEditable)
+    )
+  },
+  passphrase: {
+    required: helpers.withMessage(
+      tc('exchange_keys_form.passphrase.non_empty'),
+      requiredIf(
+        computed(() => get(sensitiveFieldEditable) && get(requiresPassphrase))
+      )
```

Comment:
can this be written as 
```typescript
and(requiredIf(sensitiveFieldEditable),requiredIf(requiresPassphrase));
//or
requiredIf(and(sensitiveFieldEditable, requiresPassphrase));
```

https://vuelidate-next.netlify.app/validators.html#and